src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,18 @@
+package com.wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
+
+    @GetMapping("/greeting")
+    public String greeting(@RequestParam(name = "name", required = false, defaultValue = "World") String name,
+            Model model) {
+        model.addAttribute("name", name);
+        return "greeting";
+    }
```

Comment:
Hi @JoelIvanStella, what do you think about adding constants to these strings?. It's very useful to reuse them and make it easier to modify them :wink:

Reply:
Yes Mati, it`s a good practice. I push the change ;)